src/constants.ts

```diff
@@ -298,6 +337,24 @@ interface IViewsForNetwork {
 }
 
 export const VIEWS_FOR_NETWORK: { [key: number]: IViewsForNetwork } = {
+  250: {
+    dashboard: true,
+    stake: true,
+    wrap: true,
+    zap: false,
+    threeTogether: true,
+    bonds: true,
+    network: true,
+  },
+  4002: {
+    dashboard: true,
+    stake: true,
+    wrap: true,
+    zap: false,
+    threeTogether: true,
```

Comment:
```suggestion
    threeTogether: false,
```

Reply:
@puf17640

Reply:
kept all of them (apparently not zap) to true because i experienced a weird bug where they would not show up in the sidebar

Reply:
ok let's do this first then